django_short_urls/views.py

```diff
@@ -110,54 +111,72 @@ def main(request, path):
 
 
 @require_POST
-def new(request):
+def new(request):  # pylint: disable=too-many-branches
     '''
     Create a new short url based on the POST parameters
-    '''
 
-    if 'login' in request.REQUEST and 'api_key' in request.REQUEST:
-        login = request.REQUEST['login']
-        api_key = request.REQUEST['api_key']
+    Temporary pylint disable: OPS-4755 - once the temporary switches below have been removed,
+    remove the pylint disable above.
+    '''
+    user = None
+    if 'HTTP_AUTHORIZATION' in request.META:
```

Comment:
(copied from #13)
I'm… kind of mind-boggled that Django doesn't have any basic auth facilities.

That whole (pretty generic) basic auth section is certainly worth being in a separate helper.